airbyte-integrations/connectors/destination-iceberg/build.gradle

```diff
@@ -24,24 +24,24 @@ application {
 
 dependencies {
 
-    implementation('org.apache.spark:spark-sql_2.13:3.3.2') {
+    implementation('org.apache.spark:spark-sql_2.13:3.5.0') {
```

Comment:
I would rather leave the Spark version as it is. The PR is about updating the Iceberg version, not the Spark version. Updating the Spark version might have other implications, such as behavioral feature changes